packages/adblocker/src/engine/engine.ts

```diff
@@ -271,6 +284,10 @@ export default class FilterEngine extends EventEmitter<
     }
     engine.lists = lists;
 
+    // Deserialize preprocessors
+    engine.preprocessors = PreprocessorBucket.deserialize(buffer);
+    engine.preprocessors.updateEnv(env);
```

Comment:
Not sure if we need `env` here if we serialize `excluded` as well. This way we get back the engine in the same configuration it had before serializing (but then it might require the client to explicitly call `updateEnv` if the engine is downloaded from CDN).

Reply:
We don't need 'env' here if we serialise the 'excluded' array too.

Reply:
The requirement of `env` has been removed by [`c312bf0` (#3782)](https://github.com/ghostery/adblocker/pull/3782/commits/c312bf0f5aff739a6043444c882f422920eefca8)